app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -356,6 +357,20 @@ private function updateStatus($currentStatus, $newStatus)
         return $indexOfCurrentStatus < $indexOfNewStatus ? $newStatus : null;
     }
 
+    /*
+     * Translate "Opt-in to Partner email?" from Rock the Vote CSV to a Northstar email_subscription_status
+     *
+     * @param array $sms_status
+     * @return string
+    */
+    private function transformEmailSubscriptionStatus($email_subscription_status)
+    {
+        if ($email_subscription_status === 'Yes') {
```

Comment:
Similarly -- do we need to check for any potential variations like `yes`, `Y`?

Reply:
It should be `Yes` but if it's low lift to check for at least `yes` that would more likely than it changing to `Y` on us suddenly. But again, yea, dependent on if RTV changes things

Reply:
Yeah, I feel like a `str_to_boolean` helper could be a nice safeguard here. 👍